README.md

```diff
@@ -12,37 +12,38 @@ A Grafana Backend Plugin that handles rendering panels &amp; dashboards to PNGs
 
 ### No dependencies
 
-This plugin have been packaged into a single executable together with [Node.js](https://nodejs.org/) runtime and [Chromium](https://www.chromium.org/Home) so it doesn't require any additional dependencies to be installed on the Grafana server.
+This plugin is packaged in a single executable with [Node.js](https://nodejs.org/) runtime and [Chromium](https://www.chromium.org/Home). It does not require any additional software to be installed on the Grafana server.
 
 ## Installation
 
 ### Using grafana-cli
 
-NOTE: Installing this plugin using grafana-cli is supported from Grafana v6.4.
+**NOTE:** Installing this plugin using grafana-cli is supported from Grafana v6.4.
 
 ```
 grafana-cli plugins install grafana-image-renderer
 ```
 
 ### Clone into plugins folder
 
-1. git clone into Grafana external plugins folder.
-2. Install dependencies and build
+1. Git clone this repo into the Grafana external plugins folder.
+2. Install dependencies and build.
+<!--- Wait, what? You said in the No dependencies section that there were no dependencies. --->
```

Comment:
Easier if you add comments in GitHub PR instead of code. This is something I have on my agenda to rewrite to make this more clear. Thanks for pointing out. Please remove this and we can merge.